### ISSUE TITLE
Option to require invitation except when creating an account via Oauth2?

### ISSUE THREAD

**Issue by poVoq** [loomio/loomio#10859]
I noticed that if I set `FEATURES_DISABLE_CREATE_USER=1` to prevent random people from signing up via email, this also prevents people from signing up via the Oauth2 login button. Linking to an existing account and signing in via Oauth2 works though.

It would be great if there was an option to make an exception for users already authenticated via the external Oauth2 provider.

Thanks!

**Comment by robguthrie** [comment]
I think you want to use "FEATURES_DISABLE_EMAIL_LOGIN" and remove FEATURES_DISABLE_CREATE_USER

**Comment by poVoq** [comment]
Yes that would be an option, but it would prevent users from using non-oauth logins completely.

In our setup the idea is that people that are members can use the Oauth2 signups/login with their existing external accounts, but group admins can also share secret invite links for creating non-member accounts. With the existing config options a setup like this is currently not possible I think.

**Comment by robguthrie** [comment]
What is your organization? I've noticed a few requested recently that I make Loomio suit your needs, but I'm only interested in working for customers or sponsors right now.

**Comment by poVoq** [comment]
Just a volunteer run community of FOSS game devs, so sadly we can't do any paid commissions.

**Comment by robguthrie** [comment]
Closing this as it's not on Loomio roadmap. It's a useful suggestion, but I don't want open issues I'm not going to work on.
